### ISSUE TITLE
Considering Whonix in addition to Tails?

### ISSUE THREAD

**Issue by JeremyRand** [OpenBitcoinPrivacyProject/wallet-ratings#17]
Has there been any discussion of adding a criterion for Whonix compatibility (in addition to Tails)?  I'm not sure how compatible Whonix is compared to stock Debian (on which it's based); but it seems like something that a number of privacy-conscious users would be interested in.  Thoughts?

**Comment by kristovatlas** [comment]
A bit more detail about Whonix:

It consists of two virtual machine images, primarily intended for use with VirtualBox. One is a "workstation" image, and another is a "router" image. 

> The Gateway VM is responsible for running Tor, and has two virtual network interfaces. One of these is connected to the outside Internet via NAT on the VM host, and is used to communicate with Tor relays. The other is connected to a virtual LAN that runs entirely inside the host.
> 
> The Workstation VM runs user applications. It is connected only to the internal virtual LAN, and can directly communicate only with the Gateway, which forces all traffic coming from the Workstation to pass through the Tor network. The Workstation VM can "see" only IP addresses on the Internal LAN, which are the same in every Whonix installation.
> 
> Unlike Tails (operating system), Whonix is not "amnesic"; both the Gateway and the Workstation retain their past state across reboots.

Other notes:
- The images combined are about 3.1 GB, and some setup is required. [1]
- Privacy is more prone to compromise via host malware infection than with Tails.
- Adding this to the criteria would increase the amount of work for relevant tests by 100+%.
- Whonix is based on Debian [2], so there is a fair chance that any application that doesn't work on Tails will also not work on Whonix.

Links:
[1] https://www.whonix.org/wiki/Download#Download_Whonix
[2] https://www.whonix.org/wiki/Comparison_with_Others

**Comment by JeremyRand** [comment]
> Privacy is more prone to compromise via host malware infection than with Tails.

Not necessarily; it's more of a tradeoff.  It is true that Whonix will retain malware infections across reboots, while Tails does not.  However, Tails allows malware with root access to bypass Tor, whereas Whonix additionally requires malware to break out of VirtualBox to be able to bypass Tor.

> Whonix is based on Debian [2], so there is a fair chance that any application that doesn't work on Tails will also not work on Whonix.

Yes, I think this is the biggest concern I have about adding such a criterion.  I'm not sure how divergent the compatibility is between Whonix and Tails.
